lib/import/brca/providers/nottingham/nottingham_handler.rb

```diff
@@ -5,29 +5,78 @@ module Brca
+                             'BRCA1 + BRCA2 + PALB2'                              => :full_screen,
+                             'Breast Cancer Core Panel'                           => :full_screen,
+                             'Breast Cancer Full Panel'                           => :full_screen,
+                             'Breast Core Panel'                                  => :full_screen,
+                             'BRCA1/BRCA2 PST'                                    => :targeted_mutation,
+                             'Cancer PST'                                         => :targeted_mutation }.freeze
+          # rubocop:enable Layout/LineLength
+
+          TEST_STATUS_MAP = { '1: Clearly not pathogenic' => :negative,
+                              '2: likely not pathogenic' => :negative,
+                              '2: likely not pathogenic variant' => :negative,
+                              'Class 2 Likely Neutral' => :negative,
+                              'Class 2 likely neutral variant' => :negative,
+                              '3: variant of unknown significance (VUS)' => :positive,
+                              '4: likely pathogenic' => :positive,
+                              '4:likely pathogenic' => :positive,
+                              '4: Likely Pathogenic' => :positive,
+                              '5: clearly pathogenic' => :positive,
+                              'Mutation identified' => :positive }.freeze
+
+          TEST_SCOPE_TTYPE_MAP = { 'Diagnostic' => :full_screen,
+                                   'Indirect'   => :full_screen,
+                                   'Predictive' => :targeted_mutation }.freeze
 
           PASS_THROUGH_FIELDS = %w[age authoriseddate
                                    receiveddate
                                    specimentype
                                    providercode
                                    consultantcode
-                                   servicereportidentifier] .freeze
+                                   servicereportidentifier].freeze
+
+          NEGATIVE_TEST = /Normal/i
+          VARPATHCLASS_REGEX = /(?<varpathclass>[0-9](?=:))/
+
+          CDNA_REGEX = /c\.(?<cdna>.?[0-9]+[^\s|^, ]+)/i
 
-          NEGATIVE_TEST = /Normal/i.freeze
-          VARPATHCLASS_REGEX = /(?<varpathclass>[0-9](?=\:))/.freeze
-          CDNA_REGEX = /c\.(?<cdna>[0-9]+[^\s|^, ]+)/i .freeze
+          EXON_REGEX = /ex(?<ons>[a-z]+)?\s?(?<exons>[0-9]+(?<otherexons>-[0-9]+)?)\s
+                        (?<vartype>del|dup)|(?<vartype>del[a-z]+|dup[a-z]+)(?<of>\sof)?\s
+                        exon(?<s>s)?\s(?<exons>[0-9]+(?<otherexons>-[0-9]+)?)/xi
+
+          PROTEIN_REGEX = /p\..(?<impact>.+)\)/i
 
           def initialize(batch)
             @failed_genotype_parse_counter = 0
             @genotype_counter = 0
-            @ex = LocationExtractor.new
             super
           end
 
```

Comment:
@FrancescoNea , we can remove this method as we do not change values of these variables in the handler for counts and super class initialisation does the job for us .